tdesign/src/themes/muiTheme.ts

```diff
@@ -70,7 +70,7 @@ export const muiTheme = ({
     rightMarginInsideNavProfileMenu: `max(2rem, calc((100vw - ${breakpointValues.desktop}px) / 2) + ${magicNumbers.nav.profile.fromRightOfPageToLeftOfMenu})`,
 
     brandGradient: `linear-gradient(90deg, rgb(249 69 105 / 100%) 0%, rgb(255 128 153 / 50%) 100%)`,
-    pinkBackgroundGradient: `linear-gradient(185.2deg, #FDF2F4 24.37%, rgba(253, 242, 244, 0) 103.81%)`,
+    pinkBackgroundGradient: `linear-gradient(180deg, #FDF2F4 0%, #FDF3F5 100%)`,
```

Comment:
The gradient looks pretty solid to me :smile: By solid I mean that I cannot see the color change in any way. Is that expected? Looking at the hex colors, it's almost the same color

![image](https://user-images.githubusercontent.com/889383/147740540-fb3f2d38-5c96-456d-b5f1-ace5fb96d2e1.png)

Did you take it straight from Figma?

Reply:
Yes, Kush and I noticed that also, we actually hopped on a brief call about exactly this. 
This gradient is very subtle, almost to the point where you can't see it, but it's there and different from the previous.

There is a Figma component this came from, I'm not sure where it is at the moment but Kush may remember if you're trying to track it down.

(FYI, historically there were also a couple cases where a "gradient" had both colors set to identical values, which is a noop. I think those have been resolved though)

Reply:
> There is a Figma component this came from, I'm not sure where it is at the moment but Kush may remember if you're trying to track it down.

Adding it to the PR's description / ClickUp ticket would certainly make it easier to find :slightly_smiling_face: I tracked it down and added it to the ClickUp ticket.

To me, it does not look like we should be using a gradient at all in the CSV preview/ingestion flow. The background behind the public/private header on the CSV preview pager looks like a solid color to me. The fact that it uses a gradient for the background is basically a hack that makes the background solid.

Looking at the definition of the gradient in Figma, I see:

```css
linear-gradient(185.2deg, #FDF2F4 24.37%, rgba(253, 242, 244, 0) 103.81%);
```

which is exactly the previous value. I suggest reverting this commit and not using `pinkBackgroundGradient` at all in the CSV preview.